lms/templates/instructor/instructor_dashboard_2/instructor_dashboard_2.html

```diff
@@ -116,11 +116,12 @@
         ## to keep this short, sections can be pulled out into their own files
 
         % for section_data in sections:
-          <section id="${ section_data['section_key'] }" class="idash-section">
-            <%include file="${ section_data['section_key'] }.html" args="section_data=section_data" />
+          <section id="${ section_data['section_key'] }" class="idash-section" aria-labelledby="header-${section_data['section_key']}">
```

Comment:
I don't think the accessibility checks test to ensure things match, only that proper structure and valid attributes exist. It might be something I can add to a Jasmine test though - checking for matching values, etc. I started a Jasmine test for follow-up work in AC-415 (proper tab widgets) so I could include it there, or make a smaller test file here that is added to later in that work?